tests/suite/manifests/telemetry/collector-values.yaml

```diff
@@ -1,5 +1,7 @@
 mode: deployment
 replicaCount: 1
+image:
+  repository: otel/opentelemetry-collector-k8s
```

Comment:
We want to use this image, right?

https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-collector/UPGRADING.md#0880-to-0890

Reply:
I believe so